torch/nn/functional.py

```diff
@@ -1931,6 +1931,18 @@ def hardsigmoid(input: Tensor, inplace: bool = False) -> Tensor:
         return torch._C._nn.hardsigmoid_(input)
     return torch._C._nn.hardsigmoid(input)
 
+bias = _add_docstr(
+    torch._C._nn.bias,
```

Comment:
And from reading the native_functions read me this the entrypoint to at::native::bias